tds/src/main/java/thredds/server/ncss/controller/NcssGridController.java

```diff
@@ -164,8 +164,9 @@ private File makeCFNetcdfFile(CoverageCollection gcd, String responseFilename, N
 
     // Test maxFileDownloadSize
     long maxFileDownloadSize = ThreddsConfig.getBytes("NetcdfSubsetService.maxFileDownloadSize", -1L);
-    if (version == NetcdfFileFormat.NETCDF4)
+    if (version.isNetdf4format()) {
```

Comment:
nice! :)

Not really related to this PR but think there is a typo in that function, its should be "Netcdf" instead of "Netdf" 😂